contracts/InterruptibleInverseVester.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.3;
+import "./InverseVester.sol";
```

Comment:
For clarity purposes I suggest adding an additional import line for `IInv` or to switch to the `import { ... } from "..."` syntax:
```typescript
import {
    IInv,
    InverseVester
} from "./InverseVester.sol";
```